packages/core/src/hooks/useContractFunction.ts

```diff
@@ -56,7 +57,7 @@ export function connectContractToSigner(contract: Contract, options?: Options, l
 
 export function useContractFunction(contract: Contract, functionName: string, options?: Options) {
   const [state, setState] = useState<TransactionStatus>({ status: 'None' })
-
+  const { addTransaction } = useContext(TransactionsContext)
```

Comment:
Why not use:
`useTransactions()`

Reply:
It's actually from a previous pull request, I forgot about these hooks, will change it in another pr.